mrc-475 testdata update

r-ash opened the pull request:
This PR will

* Update testdata to data expected for users to upload and fix tests
* Update build scripts to include packages required by naomi

In particular this updates the format of the data and fixes any validation to support that. The `.geojson` update is required for returning the set of region filters.
The only anticipated change to the format of this data is to replace `age_group_id` to return human readable values for ID instead of numeric values so it is easier for the users uploading the data.

richfitz commented on .travis.yml:
```diff
@@ -18,12 +18,15 @@ services:
 before_script:
   - R CMD INSTALL .
 cache: packages
+repos:
```
do you want to try using `Additional_repositories` in `DESCRIPTION` for this? I'm not sure I see how this is being used as-is in the travis build as it's not listed as a dependency (though I do see it in the dockerfile)

r-ash replied:
So this is only in there because it is requried by naomi and we're including naomi in the build. I think there isn't a way for naomi to automatically install that for us?

richfitz replied:
If naomi has an additional repos listed in *its* DESCRIPTION I believe it will work ok

r-ash replied:
I'll make a note to update that tomorrow - I was handling it through travis like I was here.

richfitz commented on tests/testthat/test-endpoints-validate-survey-and-programme.R:
```diff
@@ -57,7 +57,10 @@ test_that("endpoint_validate_survey_programme supports ANC file", {
   ## Sanity check that data has been returned
   expect_true(length(response$data$data) >= 800)
-  expect_equal(typeof(response$data$data[[1]]$value), "double")
+  ## TODO: Add test to ensure we are returning prevalence & art coverage are
```
Does this have a corrsponding ticket?

r-ash replied:
Added the ticket no to the comment.

richfitz commented on R/validate_inputs.R:
```diff
@@ -145,10 +145,14 @@ do_validate_anc <- function(anc, shape) {
   assert_consistent_regions(read_regions(shape, "shape"),
                             read_regions(anc, "anc"),
                             "ANC")
+  ## TODO: Call naomi to calculate prevalence & art_coverage which is what we
```
Does this have a corresponding ticket?

r-ash replied:
Added the ticket no to the comment.

r-ash commented:
Added a README about the testdata now. Mostly this is coming from Jeff - though I don't know what the source of that is. Let me know if you think I should chase that up further. I'm not expecting any updated outside of known ones which are coming which I've listed in the README. 

I've also added task ids to the TODO tasks.
